src/backend/distributed/commands/statistics.c

```diff
@@ -445,19 +446,30 @@ GetExplicitStatisticsCommandList(Oid relationId)
 
 		if (alterStatisticsOwnerCommand != NULL)
 		{
-			alterStatisticsCommandList =
-				lappend(alterStatisticsCommandList,
+			explicitStatisticsCommandList =
+				lappend(explicitStatisticsCommandList,
 						makeTableDDLCommandString(alterStatisticsOwnerCommand));
 		}
 	}
 
+	/* find indexes on current relation, in case of modified stats target */
+	Relation relation = relation_open(relationId, AccessShareLock);
+	List *indexOidList = RelationGetIndexList(relation);
+	relation_close(relation, AccessShareLock);
+
+	Oid indexOid = InvalidOid;
+	foreach_oid(indexOid, indexOidList)
```

Comment:
minor: to prevent mixed usage of `oid` & `id`, can you go over `Oid`/ `oid` usages for this pr and convert them to `id` ones ?

Reply:
seems not addressed yet, unresolving

Reply:
it's addressed. probably your browser didn't refresh